### ISSUE TITLE
Is it possible to batch labeling(label multiple images at the same time)? 

### ISSUE THREAD

**Issue by wanburana** [HumanSignal/label-studio#1239]
**Is your feature request related to a problem? Please describe.**
Me and my team design the data collection (activity classification) strategy that make same class close to each other, it's also relatively easy to identify which class it is. 

Is it possible to label multiple images at the same time, going through each image is kinda time consuming considering we get a lot of image. I didnt see that option anywhere in the setting.

**Describe the solution you'd like**
And option to batch label images, texts, etc.

**Comment by makseq** [comment]
Hi, we had some experimental features where you could propagate one annotation from one task to many another tasks. Will it solve your problem somehow?

**Comment by wanburana** [comment]
> We had some experimental features where you could propagate one annotation from one task to many another tasks. Will it solve your problem somehow?

I'm not sure that's what I intended to solve, I want to be able to either:
- select multi images and assign to one class in a click (without having to go through image one by one) or
- left click drag to select multiple images and assign to one class 


(I understood that it's kind of defeat the purpose of using annotation tool, but I think my images is relatively easy to classify without having to look into detail, and I want to convince my co-worker that it is better than drag-and-drop on typical Windows's folder). Could you provide the link for more info? I will take a look to see if it can be adapted to my problem or not.

**Comment by makseq** [comment]
https://label-studio.slack.com/archives/C01SKFX54QK/p1626423069150600 (there were more similar questions in slack)

**Comment by Dario-Mantegazza** [comment]
I have a similar problem, I'm trying to find a fast way to label multiple images that are similar to each other. I would like to do a shift +right click and select n rows and for those tasks set the same classification label. I've also commented on the same problem here https://github.com/heartexlabs/label-studio/issues/1667#issuecomment-1068075840

**Comment by makseq** [comment]
You can solve it using the experimental "Propagate annotations" feature, as it's mentioned in the related issue.

**Comment by Dario-Mantegazza** [comment]
True, but what I was trying to highlight is the lack of drag-select or shift+right click selection both in the grid and table view. I think that these two are two separate functions that combined will solve both this and the other issue.

**Comment by makseq** [comment]
@Dario-Mantegazza Could you please create a new issue with shift + drag feature request?

**Comment by Jojodike** [comment]
Hi, I have the same issue too. I have a bunch of images and I wanted to know if there's a way for me to do a batch labeling of each image in my dataset that are of the same class.

**Comment by Jojodike** [comment]
Hi, I have the same issue too. I have a bunch of images and I wanted to know if there's a way for me to do a batch labeling of each image in my dataset that are of the same class.

**Comment by makseq** [comment]
@Jojodike 
We have the experimental prototype - "Propagate annotations" in Data Manager actions. 

1. Run LS with `EXPERIMENTAL_FEATURES=1` environment variable
2. Submit the first annotation, remember its ID. 
3. Go to data manager full view or grid view.
4. Select tasks you want to label.
5. Call action "Propagate annotations" via Data Manager actions. 
6. Specify annotation ID you want to propagate to other annotations. 

![image](https://user-images.githubusercontent.com/501780/214993982-3ebd835c-d1da-46c1-bb5a-a34bb0c951e7.png)

**Comment by AbubakarSaad** [comment]
Hello,

We also do support bulk labeling now: https://docs.humansignal.com/guide/labeling_bulk.html
